groovy/gradle.java/src/org/netbeans/modules/gradle/java/classpath/GradleSourcesImpl.java

```diff
@@ -162,13 +164,7 @@ public GradleSourcesImpl(Project proj) {
                     }
                 }
             }
-            Collections.sort(ret, new Comparator<SourceGroup>() {
-
-                @Override
-                public int compare(SourceGroup o1, SourceGroup o2) {
-                    return o1.getName().compareTo(o2.getName());
-                }
-            });
+            Collections.sort(ret, (SourceGroup o1, SourceGroup o2) -> o1.getName().compareTo(o2.getName()));
```

Comment:
```suggestion
            Collections.sort(ret, Comparator.comparing(SourceGroup::getName));
```

or something like that (look it up).

Reply:
Thanks! Learn a new thing every day!
The test will be a new PR as it is not that trivial to create one for this.